src/main/java/org/folio/des/config/FolioExecutionContextHelper.java

```diff
@@ -32,18 +31,10 @@ public class FolioExecutionContextHelper {
   private final AuthService authService;
   private final SecurityManagerService securityManagerService;
   private boolean registered = false;
-
-  private final Map<String, Map<String, Collection<String>>> okapiHeaders = new ConcurrentHashMap<>();
-
-  public void storeOkapiHeaders() {
-    if (MapUtils.isNotEmpty(folioExecutionContext.getOkapiHeaders())) {
-      log.info("Got OKAPI headers.");
-      okapiHeaders.put(folioExecutionContext.getTenantId(), folioExecutionContext.getOkapiHeaders());
-    }
-  }
+  // TODO find a way to get current 'x-okapi-url', might be list of urls
+  private final String okapiUrl = "http://10.0.2.15:9130";
```

Comment:
Let's take it from the env variable and set some default

Reply:
Thank you, fixed.